mangaki/mangaki/templates/mangaki/reco_list.html

```diff
@@ -11,29 +11,13 @@
             Notez-les puis rechargez la page pour en faire apparaître
             d'autres.
 
-    Inclure la liste des "À voir" dans les recommandations : <input type="checkbox" name="reco_willsee_ok" {% if user.profile.reco_willsee_ok %} checked{% endif %}/>
```

Comment:
I'm happy to see this leave, but can we replace it with some short text explaining how people should proceed to get recommendations from their "To watch" list? (I think this is your plan for a later PR)